project/profiles/capacity4more/behat/features/bootstrap/FeatureContext/Node.php

```diff
@@ -62,12 +62,13 @@ public function iShouldNotBeAllowedToCreateA($type) {
   }
 
   /**
-   * @Then /^I should be allowed to create a "([^"]*)"$/
+   * @Then /^I should be allowed to create a "([^"]*)" in group "([^"]*)"$/
    */
-  public function iShouldBeAllowedToCreateA($type) {
+  public function iShouldBeAllowedToCreateA($type, $group) {
+    $machine_readable_group = strtolower(preg_replace('@[^a-zA-Z0-9_]+@', '-', trim($group)));
```

Comment:
@ybaras 
It seems the same as the other one,  maybe it'll be better to take out to an external function ?